src/test/java/Homework20Test.java

```diff
@@ -0,0 +1,19 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.*;
+
+public class Homework20Test extends BaseTest {
+    @Test (dataProvider = "LoginAndPlaylistExample", dataProviderClass = BaseTest.class)
```

Comment:
Please, delete `throws InterruptedException`

Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.

Reply:
resolved

Reply:
Hello

Please, re-submit homework 20 for review